node-graph/interpreted-executor/Cargo.toml

```diff
@@ -13,7 +13,7 @@ quantization = ["graphene-std/quantization"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-graphene-core = { path = "../gcore", features = ["async", "std"] }
+graphene-core = { path = "../gcore", features = ["std"] }
```

Comment:
Should the other deps in this file (lower down) be switched over to `workspace` too?